RPLCD/pigpio.py

```diff
@@ -316,6 +332,10 @@ def _send(self, value, mode):
         # Switch on pigpio's exceptions
         pigpio.exceptions = True
 
+        # Record the time for the tail-end of the last send event
+        if self.compat_mode:
+            self.last_send_event = now()
```

Comment:
It's a bit weird that the compat mode is something that is supported in the base class, but that implementation details need to be handled in the subclass. How about this?

- Add two functions in the base class called `_compat_mode_wait()` and `_compat_mode_record_send_event()`.
- In the two subclasses, never explicitly check for `self.compat_mode`, instead simply call those two functions (which are a no-op if compat mode is not enabled).

Another benefit of this is that the Python 2 compat importing of `now` is only needed once (in the base class).

Reply:
Yup, done